src/sdl/SDL.cpp

```diff
@@ -323,23 +323,29 @@ bool sdlCheckDirectory(const char* dir)
 
 char* sdlGetFilename(char* name)
 {
-    char path[1024] = ""; // avoid warning about uninitialised value
+    char path[1024];
     char *filename = strrchr(name, FILE_SEP);
     if (filename)
-        strncpy(path, filename + 1, strlen(filename));
+        strcpy(path, filename + 1);
```

Comment:
What was wrong with the strncpy/sprintf versions?

Reply:
They both do the same thing as `strcpy` only with more overhead and, in the `strncpy`'s case, a compiler warning.